guide/README.md

```diff
@@ -23,5 +23,9 @@ If you don't know JS but would like to learn about it, here are a few links to h
 * [Nodeschool, for both JavaScript and Node.js lessons](https://nodeschool.io/)
 * [MDN's JavaScript guide and full documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Google, your best friend](https://google.com)
+* [Grasshopper, a JavaScript learning app by Google](https://grasshopper.app)
+* [w3schools' free tutorials for both server-side and client-side JavaScript](https://w3schools.com)
+* [Learn-js.org's interactive tutorials](https://learn-js.org)
```

Comment:
:x: from the three sample topics i read seems to encourage bad code practices for the sake of tutorials covering a topic